apps/web/playwright/manage-booking-questions.e2e.ts

```diff
@@ -246,14 +246,18 @@ test.describe("Manage Booking Questions", () => {
         },
         select: {
           id: true,
+          name: true,
         },
       });
 
       const teamId = team?.id;
       const webhookReceiver = await addWebhook(undefined, teamId);
 
       await test.step("Go to First Team Event", async () => {
-        const $eventTypes = page.locator("[data-testid=event-types]").nth(1).locator("li a");
+        await page.getByTestId(`horizontal-tab-${team?.name}`).click();
+        await page.waitForLoadState("networkidle");
+
+        const $eventTypes = page.locator("[data-testid=event-types]").locator("li a");
```

Comment:
Instead of doing networkidle waiting, better could be to wait for selector for a certain number of seconds.

Something like page.waitForSelector('', {timeout:1000})

Reply:
We already have many places where we're doing this:

https://github.com/calcom/cal.com/blob/e3b478f5b13e44fe9a7630e74c44af47e213a849/apps/web/playwright/managed-event-types.e2e.ts#L255-L260

Reply:
Just need to note the endpoint we're actually calling